Updated Readme with Heroku info (#258)
RailsGirlsCPH/mentor-mentee-platform: server/README.md

You can use swagger to test the functionality of the currently available API's. 
You can find more information on the site: https://swagger.io/solutions/api-documentation/

For Back End Users:

Run the following commands to seed the test database & start the local server: 

```
rails db:seed
bundle exec rails server
```
Then visit the local swagger site http://localhost:3000/api-docs/index.html

For Front End Users: 

> For those on the front end, swagger docs for the api can be found at https://mentor-mentee-platform.herokuapp.com/api-docs/index.html

Reviewer: ```suggestion
For those on the front end, consult the [published Swagger API docs](https://mentor-mentee-platform.herokuapp.com/api-docs/index.html).
```

Reviewer: We have the full power of hyperlinks at our disposal! 🌟

Reviewer: Come to think of it, perhaps the section for the frontend audience should go _first_, so that they can skip the rest of the detail.

## Deployment

Using Heroku to host production instance. 

Once logged into Heroku code can be deployed by running: 

```
git push heroku whatever_branch_I_am_on_branch:master
```
Note that you cannot push to any branch other than master on Heroku, although the branch on the local machine can be anything. 

Because our project folder contains two sub projects in different languages, deployment requires setting the folder to be deployed to production in the buildpack. 

Instructions for getting around this issue were taken from this tutorial: https://elements.heroku.com/buildpacks/techgaun/heroku-buildpack-subdir-to-root

The key commands to be executed in route directory of sub-project (in my case /server) are: 

```
heroku buildpacks:add --index 1 https://github.com/techgaun/heroku-buildpack-subdir-to-root.git --app mentor-mentee-platform

> heroku config:set PROJECT_RELATIVE_PATH=packages/adminWeb --app mentor-mentee-pkatform

Reviewer: ```suggestion
heroku config:set PROJECT_RELATIVE_PATH=packages/adminWeb --app mentor-mentee-platform
```

(I guessed this was a typo.)

```

> After you deploy the app, you need to seed the database in order for swagger docs to work. 

Reviewer: This section made me think of https://devcenter.heroku.com/articles/release-phase - a Heroku feature.

As a future improvement, perhaps we can use that, for tasks to always run just before being done with deployment.

Author: Added Issue #259  to automate running tasks when deploying in Heroku.
